docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>I think the title is too long, would drop <code>and metrics</code>as: Tracking Progress in Sinkhorn and Low-Rank Sinkhorn</p><p>Also please link to Sinkhorn and LRSinkhorn here.</p><p>In general, wouldn't capitalize Low-rank, unless in a title.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Sure, done.</p>